### PR TITLE
Removed Mage_Backup

### PR DESCRIPTION
**Pull request by sreichel**
### Description (*)

Removed `Mage_Backup` code.

Can be reinstalled with `composer require openmage/module-mage-backup`

### Related Pull Requests

- See OpenMage/magento-lts#2259
- See OpenMage/magento-lts#2782

### Questions or comments

Repo should be moved to OM space.

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All automated tests passed successfully (all builds are green)
 - [x] Add yourself to contributors list

### CODE REVIEW

**Comment by fballiano** [comment]
but what about my pr? 🥺🥺

**Comment by fballiano** [comment]
> but what about my pr? 🥺🥺

ah right, it was broken.

well, as everybody knows I'm all for this.

**Review by fballiano — APPROVED** [review]

**Comment by ADDISON74** [comment]
Here #2024 and here #2259 we discussed this module. @fballiano said that there are big issues in the code of this module that the Magento team no longer wanted to solve, other than the data security ones shown by me.

I personally agree with removing it and installing it from a repository by anyone who is interested. I would go a little further, if we remove the code no one can create PRs from now on. Wouldn't it be better to create a new OpenMage repository here on GitHub so that any contribution can be made and we know about it? Those who are interested can install it either with Composer or put it back by downloading the archive and copying the files in OpenMage installation.

**Comment by fballiano** [comment]
the repo, at the moment, is https://github.com/sreichel/openmage-module-mage-backup :-)

**Comment by ADDISON74** [comment]
I know but we should have it as a branch in OpenMage.

**Comment by sreichel** [comment]
@ADDISON74 aggreed, but i cant add repos to OpenMage space.

**Comment by fballiano** [comment]
I've created https://github.com/OpenMage/module-mage-backup for you

**Comment by sreichel** [comment]
Why i cant do this?

Seems i cannot push.

**Comment by fballiano** [comment]
I don't know why you can't do that but anyway I've invite you as maintainer for that repo, but it seems the invite was accepted yes

Screenshot 2022-12-18 alle 16 54 45

**Comment by sreichel** [comment]
Thanks. Its there.

**Comment by fballiano** [comment]
did you change the packagist references too? can you? (I cannot)

**Comment by sreichel** [comment]
yes. moved both

**Review by fballiano — APPROVED** [review]

**Comment by ADDISON74** [comment]
Before approving it, I would like to ask you what happens with **Tools** branch in the Backend > System menu. The "Backups" and "Compilation" links have been grouped here. The latter has been removed from OpenMage for some time, the former via this PR. Under these conditions, System > Tools should no longer exist in OpenMage and it can be removed, I guess through another PR.

**Comment by fballiano** [comment]
> Before approving it, I would like to ask you what happens with **Tools** branch in the Backend > System menu. The "Backups" and "Compilation" links have been grouped here. The latter has been removed from OpenMage for some time, the former via this PR. Under these conditions, System > Tools should no longer exist in OpenMage and it can be removed, I guess through another PR.

I think it would disappear but it wouldn't be a problem because it reappears automatically in case a module (that uses it) gets installed

**Comment by sreichel** [comment]
@ADDISON74 "tools" should not be visible when Mage_Backup is disabled (default). If you enable it "tools" appears.

File: README.md [hunk 1]
```diff
@@ -111,7 +111,18 @@ Most important changes will be listed here, all other changes since `19.4.0` can
...
+  - `Mage_Backup`
+  - `Mage_Compiler`
+  - `Mage_GoogleBase`
+  - `Mage_Xmlconnect`
+  - `Phoenix_Moneybookers`
+
+`Mage_Backup` was disabled by Magento an has removed in <VERSIONUMBER>. If you still want to use it:
+
+```
+composer require openmage/bc-mage-backup:dev-master
```
Comment (Flyingmana) [inline README.md:124]: using `master` is an outdated deprecated naming pattern for branches. Common alternative is `main`

**Review by Flyingmana — DISMISSED** [review]
Replacement module should have at least one valid Release, just in case we need to do a bugfix or bc breaking release.
Version 1.0.0 would be alright.

PS: I cant do this myself as I did not get permissions assigned for the related repository

File: README.md (hunk 1 above)
Comment (sreichel) [reply README.md:124]: Dont have permissions too.

README will be updated soon.

Comment (fballiano) [reply README.md:124]: Added you both as admins

Comment (Flyingmana) [reply README.md:124]: created directly the 1.0.0 version

**Review by fballiano — DISMISSED** [review]

**Comment by ADDISON74** [comment]
With the default installation there are 3 modules having the output disabled Mage_Backup, Mage_Poll, Mage_Tag. Sven is right when the Mage_Backup module is disabled from XML configuration or its output in Backend the Tools link doesn't appear in System menu.

If you have made sure that you have identified all the files of this module I don't see what would prevent us from approving it.

**Comment by sreichel** [comment]
> With the default installation there are 3 modules having the output disabled Mage_Backup, **Mage_Poll, Mage_Tag**.

@Flyingmana @fballiano can you please create repos for `Mage_Poll` and `Mage_Tag` or give permission? :sunglasses:

**Comment by fballiano** [comment]
I can't give you that permission but I made you admin for https://github.com/OpenMage/module-mage-poll and https://github.com/OpenMage/module-mage-tag

**Review by luigifab — DISMISSED** [review]

**Review by fballiano — DISMISSED** [review]

**Comment by sreichel** [comment]
Yeah ... conflicts on a file to delete ... :/

**Review by ADDISON74 — APPROVED** [review]

**Review by fballiano — APPROVED** [review]
I don't understand how this is not considered BC while https://github.com/OpenMage/magento-lts/pull/2863 is, but they're both ok in my opinion

**Comment by sreichel** [comment]
> Dont think its a breaking change. It should be included in download and everyone who uses composer can install it when required. Just needs some documentation.

**Comment by fballiano** [comment]
you know what I was referring to

**Pull request merged**